README.md

```diff
@@ -7,54 +7,32 @@
 [![Quality Score](https://img.shields.io/scrutinizer/g/graze/skeleton-project.svg?style=flat-square)](https://scrutinizer-ci.com/g/graze/skeleton-project)
 [![Total Downloads](https://img.shields.io/packagist/dt/graze/skeleton-project.svg?style=flat-square)](https://packagist.org/packages/graze/skeleton-project)
 
->This is heavily inspired by the skeleton project from [The PHP League](https://github.com/thephpleague/skeleton). With some changes based on our preferences. Thanks to them!
+> This is heavily inspired by the skeleton project from [The PHP League](https://github.com/thephpleague/skeleton).
+> With some changes based on our preferences. Thanks to them!
```

Comment:
This entire block quote is now a duplicate of what's in `dos/index.md`. Would it be better to just reduce this section to just pointing people to that page and then telling them to delete this block once they have set everything up?